testing/src/main/java/org/oppia/android/testing/media/TestMediaPlayerModule.kt

```diff
@@ -0,0 +1,22 @@
+package org.oppia.android.testing.media
+
+import dagger.Module
+import dagger.Provides
+import org.oppia.android.testing.IsOnRobolectric
+import javax.inject.Provider
+
+/**
+ * Dagger [Module] that provides TestMediaPlayer.
+ */
+@Module
+class TestMediaPlayerModule {
+
+  @Provides
+  fun provideTestMediaPlayer(
+    @IsOnRobolectric isOnRobolectric: Boolean,
```

Comment:
Should we separate out `isOnRobolectric` from `TestCoroutineDispatcherModule`?

Reply:
SGTM.

Reply:
Created a separate issue for this. Could be a variety for contributors. 
https://github.com/oppia/oppia-android/issues/2021